src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -175,10 +165,9 @@ public async Task GetInfoPackage()
             HttpContext.Response.Headers["Content-Disposition"] = contentDisposition;
             HttpContext.Response.Headers["Content-Type"] = "application/zip";
 
-
-            var token = CreateOperationToken();
-
             var operationId = GetLongQueryString("operationId", false) ?? ServerStore.Operations.GetNextOperationId();
+            var timeout = TimeSpan.FromSeconds(GetIntValueQueryString("timeoutInSec", false) ?? _defaultTimeoutForEachNodeInSec);
```

Comment:
Not needed here, we are going to use it only for the cluster wide

Reply:
Ok. Done.